src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/AdapterUtilCommon.cs

```diff
@@ -0,0 +1,1202 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can this file name stay "AdapterUtil"? Any reason to call it AdapterUtilCommon?

Reply:
If there are AdapterUtil.cs in the specific projects it would be `AdapterUtil.common.cs` following @DavoudEshtehari 's logic in https://github.com/dotnet/SqlClient/issues/1261#issuecomment-933838859

Reply:
Apparently, I missed the dot. 😉